### DISCUSSION TITLE
Welcome to AniwatchPlus Discussions!

### DISCUSSION THREAD

**Discussion by Serraniel** [Serraniel/AniwatchPlus#142] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sirishp3** [comment]
I was wondering if Aniwatch would ban my account because of this extension because that's what kissanime did?

**Reply by Serraniel** [reply]
Sorry for the late reply. Currently the extension is 'accepted' by the developers as we do not provide any features which is against their tos. Adding features which would violate their tos (like download feature) is also not planned for the future so I also do not see a risk this extension could get you banned in the future.

You can read up a short statement by the developers in their community discord here: https://discord.com/channels/541683465599451136/656809893323341825/804252764774989864
